tests/wpt/metadata/html/dom/interfaces.https.html.ini

```diff
@@ -6993,18 +6915,9 @@
   [HTMLMediaElement interface: new Audio() must inherit property "ended" with the proper type]
     expected: FAIL
 
-  [HTMLMediaElement interface: new Audio() must inherit property "autoplay" with the proper type]
-    expected: FAIL
-
   [HTMLMediaElement interface: new Audio() must inherit property "loop" with the proper type]
     expected: FAIL
```

Comment:
The expectations of these tests did not change. I do not know why?

Reply:
We don't support those properties yet.